python_modules/dagster-webserver/dagster_webserver/webserver.py

```diff
@@ -298,6 +300,11 @@ def build_routes(self):
                     "/logs/{path:path}",
                     self.download_captured_logs_endpoint,
                 ),
+                Route(
+                    "/dagster-webserver/notebook",
```

Comment:
dagster-webserver prefix feels odd / redundant here

Reply:
I agree. I was just cargo-culting the older format (/dagit/notebook), not knowing if it was like that for a reason. But if you don't think so, I'll drop it.